nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+// Parse URL-encoded bodies (as sent by HTML forms)app.use(express.urlencoded({ extended: true }));// Parse JSON bodies (as sent by API clients)app.use(express.json());
+app.use("/calculator", router);
+router.get("/:operator", async (request, response) => { 
```

Comment:
I see what you tried to do here :) 
I think though that it'd be cleaner if you just had created different routes for each operation. eg.:
```
app.get("/calculator/add", (req, res)
app.get("/calculator/subtract", (req, res)
...
```